apps/bfd-server/bfd-server-docs/src/main/java/gov/cms/bfd/datadictionary/CsvToExcelCommand.java

```diff
@@ -0,0 +1,326 @@
+            c.setCellStyle(style);
+          }
+        });
+  }
+
+  /**
+   * Apply a consumer function to a range of cells in a sheet.
+   *
+   * @param sheet the sheet to update
+   * @param range the range of cells to update
+   * @param fn a consumer function that takes a cell and returns void
+   */
+  private static void applyToRange(Sheet sheet, CellRangeAddress range, Consumer<Cell> fn) {
+    for (int i = range.getFirstRow(); i <= range.getLastRow(); i++) {
+      var row = sheet.getRow(i);
+      if (row != null) {
+        for (int j = range.getFirstColumn(); j <= range.getLastColumn(); j++) {
+          var cell = row.getCell(j);
+          if (cell != null) {
+            fn.accept(cell);
+          }
+        }
+      }
+    }
+  }
+
+  /**
+   * Create the style for the sheet header row.
+   *
+   * @param wb the workbook
+   * @return a CellStyle
+   */
+  private static CellStyle getHeaderCellStyle(Workbook wb) {
+    var style = wb.createCellStyle();
+    style.setFillForegroundColor(new XSSFColor(new java.awt.Color(67, 133, 244), null));
+    style.setFillPattern(FillPatternType.SOLID_FOREGROUND);
+    var font = wb.createFont();
+    font.setBold(true);
+    font.setColor(IndexedColors.WHITE.getIndex());
+    style.setFont(font);
+    style.setVerticalAlignment(VerticalAlignment.TOP);
+    style.setWrapText(true);
+    return style;
+  }
+
+  /**
+   * Create the style for the sheet data rows.
+   *
+   * @param wb the workbook
+   * @return a CellStyle
+   */
+  private static CellStyle getDataCellStyle(Workbook wb) {
+    var style = wb.createCellStyle();
+    style.setVerticalAlignment(VerticalAlignment.TOP);
+    style.setWrapText(true);
+    return style;
+  }
+
+  /**
+   * Create the style for shadded rows.
+   *
+   * @param wb the workbook
+   * @return a CellStyle
+   */
+  private static CellStyle getRowShadingStyle(Workbook wb) {
+    var style = wb.createCellStyle();
+    style.cloneStyleFrom(getDataCellStyle(wb));
+    style.setFillBackgroundColor(IndexedColors.GREY_25_PERCENT.getIndex());
+    style.setFillForegroundColor(new XSSFColor(new java.awt.Color(243, 243, 243), null));
+    style.setFillPattern(FillPatternType.SOLID_FOREGROUND);
+    style.setBorderTop(BorderStyle.THIN);
+    style.setTopBorderColor(IndexedColors.GREY_25_PERCENT.getIndex());
+    style.setBorderBottom(BorderStyle.THIN);
+    style.setBottomBorderColor(IndexedColors.GREY_25_PERCENT.getIndex());
+    style.setBorderLeft(BorderStyle.THIN);
+    style.setLeftBorderColor(IndexedColors.GREY_25_PERCENT.getIndex());
+    style.setBorderRight(BorderStyle.THIN);
+    style.setRightBorderColor(IndexedColors.GREY_25_PERCENT.getIndex());
+    return style;
+  }
+
+  /**
+   * Main method to create and save a new data dictionary Excel workbook.
+   *
+   * @param args the pathnames for the v1, v2 CSV input files, and the Excel workbook output file
+   */
+  public static void main(String[] args) {
+
+    if (args.length != 3) {
```

Comment:
The structure of this class is excellent.  Logic broken down into short, simple methods that compse nicely.

One nit, I like to order methods by decreasing access level.  So public at the top, then protected, then package, then private.